src/constants/models.ts

```
@@ -101,17 +168,69 @@ export const FIXTURE_CONFIG: Record<ComponentType, FixtureConfig> = {
   Door: AVAILABLE_MODELS.find(m => m.name === 'Door')!,
   Radiator: AVAILABLE_MODELS.find(f => f.name === 'Radiator')!,
   Toilet: AVAILABLE_MODELS.find(f => f.name === 'Toilet')!,
-
+  Bath: AVAILABLE_MODELS.find(f => f.name === 'Bath')!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)